packages/lesswrong/components/common/LWTooltip.tsx

```diff
@@ -24,12 +24,20 @@ interface LWTooltipProps extends ExternalProps, WithStylesProps, WithHoverProps
 
 const LWTooltip = ({classes, children, title, placement="bottom-start", hover, anchorEl, stopHover, tooltip=true, flip=true}: LWTooltipProps) => {
   const { LWPopper } = Components
+  const { hover, everHovered, anchorEl, stopHover, eventHandlers } = useHover({
```

Comment:
Currently this is throwing errors due to duplicate declarations between the props and useHover deconstruction, which has me worried you haven't tested it.

Reply:
(I think there is some chance there's a weird conflict between this branch and a local branch of mine since it has a nondescript name)